core/templates/pages/exploration-editor-page/editor-tab/state-name-editor/state-name-editor.component.html

```diff
@@ -10,7 +10,7 @@
 
       <span ng-if="$ctrl.StateNameService.isStateNameEditorShown()">
         <form class="form-horizontal" role="form" ng-submit="$ctrl.saveStateNameAndRefresh($ctrl.tmpStateName)">
-          <input type="text" ng-model="$ctrl.tmpStateName" focus-on="stateNameEditorOpened" class="protractor-test-state-name-input">
+          <input type="text" ng-model="$ctrl.tmpStateName" maxlength="50" focus-on="stateNameEditorOpened" class="protractor-test-state-name-input">
```

Comment:
Can we assign this value to a constant and use that here?

Reply:
Done!